public/index.html

```diff
@@ -23,13 +23,13 @@
     <meta name="msapplication-TileColor" content="#ffffff">
     <meta name="msapplication-TileImage" content="/favicon/ms-icon-144x144.png">
     <meta name="theme-color" content="#ffffff">
-    <meta name="description" content="A website to track your completion of a Living Pokédex">
-    <meta name="keywords" content="pokemon,pokedex,tracker,checklist,livingdex">
+    <meta name="description" content="A tool for tracking your Living Dex progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out.">
+    <meta name="keywords" content="pokemon,pokedex,track,tracker,checklist,livingdex,progress,completion">
```

Comment:
can you add "living dex" and "pokedex tracker"? or are two word phrases not allowed